python/lsst/coadd/utils/coadd.py

```diff
@@ -19,7 +19,7 @@
 # the GNU General Public License along with this program.  If not,
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
-from __future__ import division
+from __future__ import absolute_import, division
```

Comment:
Thanks! But please also append `, print_function` to make `from __future__ import absolute_import, division, print_function` if you are willing. That is our standard `from __future__` import.